src/express/commands/milestone-base.js

```diff
@@ -0,0 +1,341 @@
+/* eslint-disable dot-notation */
+import { loadDevnetConfig, splitToArray } from '../common/config-utils'
+import { timer } from '../common/time-utils'
+import { maxRetries, runCommand } from '../common/remote-worker'
+import {
+  getBlock,
+  getPeerLength,
+  createClusters,
+  getEnode,
+  fetchLatestMilestone,
+  joinAllPeers,
+  validateProposer,
+  validateFinalizedBlock,
+  checkForRewind
+} from '../common/milestone-utils'
+
+const milestoneLength = 64
+const queryTimer = (milestoneLength / 4) * 1000
+
+export async function milestoneBase() {
+  // NOTE: Make sure bor branch has logic for hardcoded primary validator
```

Comment:
So, in order to run deterministic tests for milestones, we need to make sure that in the devnet setup when we induce a reorg, there are 2 partitions where one of them has higher difficulty but less voting power and the other one has lower difficulty but more voting power. 

If we run this with the default setup of devnet, this situation may or may not arise. Hence we have to hardcode the primary validator in all the nodes so that we know which fork will have the highest difficulty. This speeds up the process of testing as we're bound to get results on every run.